pkg.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
```

Comment:
Add comment along the lines of:

Part of automatic package building for pekwm releases